### PR TITLE
Миронова -- Лабораторная работа #0

### PR DESCRIPTION
**Pull request by Anna-Mironova**

### CODE REVIEW

**Comment by Anna-Mironova** [comment]
@SemyonBevzuk , @KulyginaJulia please review

**Comment by KulyginaJulia** [comment]
:+1:

File: code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/MironovaAnnaTest.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    }
+
+    @Test
+    public void canCompareEqualComplexNumbers() {
+        ComplexNumber z1 = new ComplexNumber(1.0, 0.0);
+        ComplexNumber z2 = new ComplexNumber(1.0, 0.0);
+
+        boolean flag = z1.equals(z2);
+
+        assertTrue(flag);
```
Comment (SemyonBevzuk) [inline code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/MironovaAnnaTest.java]: I think the flag here is unnecessary.
This string is more laconically: assertTrue(z1.equals(z2));

File: code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/MironovaAnnaTest.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    }
+
+    @Test
+    public void canCompareNotEqualComplexNumbers() {
+        ComplexNumber z1 = new ComplexNumber(1.0, 0.0);
+        ComplexNumber z2 = new ComplexNumber(1.0, 1.0);
+
+        boolean flag = z1.equals(z2);
+
+        assertFalse(flag);
```
Comment (SemyonBevzuk) [inline code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/MironovaAnnaTest.java]: Similarly

**Comment by KulyginaJulia** [comment]
:+1:

**Comment by SemyonBevzuk** [comment]
👍

**Comment by Anna-Mironova** [comment]
@kirill-korniakov

**Pull request merged**